pkg/volume/gcepd/gce_util.go

```diff
@@ -185,7 +188,7 @@ func (util *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner, node *v1.
 		return "", 0, nil, "", fmt.Errorf("replication-type of '%s' is not supported", replicationType)
 	}
 
-	labels, err := cloud.GetAutoLabelsForPD(name, "" /* zone */)
+	labels, err := cloud.GetAutoLabelsForPD(disk)
```

Comment:
IIUC this is the only place where this change makes an improvement right?

Reply:
Yes, but it's called on each CreateVolume call.